x-pack/plugin/core/src/main/resources/logs-mappings.json

```diff
@@ -1,14 +1,312 @@
 {
   "template": {
     "mappings": {
+      "dynamic_templates": [
+        {
+          "match_ip": {
+            "match_mapping_type": "string",
+            "match": "ip",
+            "runtime": {
+              "type": "ip"
+            }
+          }
+        },
+        {
+          "message": {
+            "path_match": "message",
+            "mapping": {
+              "type": "match_only_text"
+            }
+          }
+        },
+        {
+          "log_level": {
+            "path_match": "log.level",
+            "mapping": {
+              "type": "keyword",
+              "ignore_above": 1024
+            }
+          }
+        },
+        {
+          "log_logger": {
+            "path_match": "log.logger",
+            "mapping": {
+              "type": "keyword",
+              "ignore_above": 1024
+            }
+          }
+        },
+        {
+          "trace_id": {
+            "path_match": "trace.id",
+            "mapping": {
+              "type": "keyword",
+              "ignore_above": 1024
+            }
+          }
+        },
+        {
+          "span_id": {
+            "path_match": "span.id",
+            "mapping": {
+              "type": "keyword",
+              "ignore_above": 1024
+            }
+          }
+        },
+        {
+          "transaction_id": {
+            "path_match": "transaction.id",
+            "mapping": {
+              "type": "keyword",
+              "ignore_above": 1024
+            }
+          }
+        },
+        {
+          "service_name": {
+            "path_match": "service.name",
+            "mapping": {
+              "type": "keyword",
+              "ignore_above": 1024
+            }
+          }
+        },
+        {
+          "service_version": {
+            "path_match": "service.version",
+            "mapping": {
+              "type": "keyword",
+              "ignore_above": 1024
+            }
+          }
+        },
+        {
+          "service_environment": {
+            "path_match": "service.environment",
```

Comment:
Before I comment on each of these, I think we need to go through each one and discuss if we really not to index it. I think we can start with a smaller subset.

Reply:
Sure, let's do that. But I don't think we should subtract a lot here.